conans/util/encrypt.py

```diff
@@ -0,0 +1,20 @@
+from base64 import urlsafe_b64encode, urlsafe_b64decode
+
+import six
+
+
+# WARNING: These are useful functions to obfuscate some data, but they are NO WAY OK FOR SECURITY!
+
+def encode(data, key):
+    assert isinstance(data, six.string_types), "Expected string type, got '{}'".format(type(data))
+    assert isinstance(key, str), "Expected 'str' type, got '{}'".format(type(key))
+    if six.PY3:
+        return urlsafe_b64encode(bytes(key + data, 'utf-8'))
+    else:
+        return urlsafe_b64encode((key.encode('utf-8') + data).encode('utf-8'))
+
+
+def decode(enc, key):
+    assert isinstance(enc, bytes), "Expected 'bytes', got '{}'".format(type(enc))
+    assert isinstance(key, str), "Expected 'str' type, got '{}'".format(type(key))
+    return urlsafe_b64decode(enc)[len(key):].decode('utf-8')
```

Comment:
This seems a bit weak, as it brings the length of the key only as the thing to "crack".  I would say it is worth to at least use something like the Vigenere-Cipher, that would require way more effort to crack, and seems quite straight forward to implement.